tests/cephfs/cephfs_mirroring/test_cephfs_mirroring_validate_cephfs-mirroring_on_multifs_setup.py

```diff
@@ -0,0 +1,575 @@
+        rc, ec = target_clients[0].exec_command(sudo=True, cmd="ceph fs ls --format json-pretty")
+        result = json.loads(rc)
+        target_clients[0].exec_command(
+            sudo=True, cmd="ceph config set mon mon_allow_pool_delete true"
+        )
+        for fs in result:
+            fs_name = fs["name"]
+            target_clients[0].exec_command(
+                sudo=True, cmd=f"ceph fs volume rm {fs_name} --yes-i-really-mean-it"
+            )
+        time.sleep(30)
+
+        log.info("Create required filesystem on Source Cluster...")
+        mds_nodes = ceph_cluster_dict.get("ceph1").get_ceph_objects("mds")
+        log.info(f"Available MDS Nodes {mds_nodes[0]}")
+        log.info(len(mds_nodes))
+        source_fs_1 = "cephfs1"
+        source_fs_2 = "cephfs2"
+        mds_names = []
+        for mds in mds_nodes:
+            mds_names.append(mds.node.hostname)
+        hosts_list1 = mds_names[-6:-4]
+        mds_hosts_1 = " ".join(hosts_list1) + " "
+        log.info(f"MDS host list 1 {mds_hosts_1}")
+        source_clients[0].exec_command(
+            sudo=True, cmd=f'ceph fs volume create {source_fs_1} --placement="2 {mds_hosts_1}"'
+        )
+        fs_util_ceph1.wait_for_mds_process(source_clients[0], source_fs_1)
+
+        hosts_list2 = mds_names[-4:-2]
+        mds_hosts_2 = " ".join(hosts_list2) + " "
+        log.info(f"MDS host list 2 {mds_hosts_2}")
+        source_clients[0].exec_command(
+            sudo=True, cmd=f'ceph fs volume create {source_fs_2} --placement="2 {mds_hosts_2}"'
+        )
+        fs_util_ceph1.wait_for_mds_process(source_clients[0], source_fs_2)
+
+        log.info("Create required filesystem on Target Cluster...")
+        mds_nodes = ceph_cluster_dict.get("ceph2").get_ceph_objects("mds")
+        log.info(f"Available MDS Nodes {mds_nodes[0]}")
+        log.info(len(mds_nodes))
+        target_fs_1 = "cephfs-rem1"
+        target_fs_2 = "cephfs-rem2"
+        mds_names = []
+        for mds in mds_nodes:
+            mds_names.append(mds.node.hostname)
+        hosts_list1 = mds_names[-4:-2]
+        mds_hosts_1 = " ".join(hosts_list1) + " "
+        log.info(f"MDS host list 1 {mds_hosts_1}")
+        target_clients[0].exec_command(
+            sudo=True, cmd=f'ceph fs volume create {target_fs_1} --placement="2 {mds_hosts_1}"'
+        )
+        fs_util_ceph1.wait_for_mds_process(target_clients[0], target_fs_1)
+
+        hosts_list2 = mds_names[-2:]
+        mds_hosts_2 = " ".join(hosts_list2) + " "
+        log.info(f"MDS host list 2 {mds_hosts_2}")
+        target_clients[0].exec_command(
+            sudo=True, cmd=f'ceph fs volume create {target_fs_2} --placement="2 {mds_hosts_2}"'
+        )
+        fs_util_ceph1.wait_for_mds_process(target_clients[0], target_fs_2)
+
+        target_user1 = "mirror_remote1"
+        target_user2 = "mirror_remote2"
+        target_site_name = "remote_site"
+
+        log.info("Enable mirroring mgr module on source and destination")
```

Comment:
Can we use for loop for whole testing hemanth. As whole process is same .
we are calling same functions twice